src/node/cli.ts

```diff
@@ -128,6 +129,7 @@ const options: Options<Required<Args>> = {
       "Disable update check. Without this flag, code-server checks every 6 hours against the latest github release and \n" +
       "then notifies you once every week that a new release is available.",
   },
+  enable: { type: "string[]" }, // There is no support for this flag.
```

Comment:
❓ How come we say "There is no support for this flag?" is that to imply that it's not supported for code-server users?

Reply:
Ah actually idk if we should keep this comment. I guess what I mean is actually "no guarantee these features will work".

Reply:
Ahhh gotcha. Maybe change it to that instead? your call!

Reply:
Maybe we could change the "Enabling features" info message to say something like:

> Enabling the following experimental features:

and at the bottom, something like:

> The code-server project does not provide stability guarantees or commit to fixing bugs relating to these experimental features. When filing bug reports, please ensure that you can reproduce the bug with all experimental features turned off.

Reply:
That's some premium copy :+1: Thanks! Will add.